.gitignore

```diff
@@ -37,3 +39,10 @@ terratest.tf.json
 
 # Ignore makefile
 ops-makefile
+
+# VisualStudioCode
+.vscode/
+
+.cache/
+
+.bash_history
```

Comment:
What's generating a .bash_history file here?

Reply:
This file was created when I was using a docker image for testing locally using bind mounts. I will remove this from .gitignore since it won't be used anymore. Good catch.